afdko/Tools/SharedData/FDKScripts/FDKUtils.py

```diff
@@ -57,22 +59,22 @@ def findFDKDirs():
 def findFDKFile(fdkDir, fileName):
 	path = os.path.join(fdkDir, fileName)
 	if os.path.exists(path):
-            return path
-        p1 = path + ".exe"
- 	if os.path.exists(p1):
-            return p1
-        p2 = path + ".cmd"
+		return path
+	p1 = path + ".exe"
+	if os.path.exists(p1):
+		return p1
+	p2 = path + ".cmd"
 	if os.path.exists(p2):
-            return p2
+		return p2
 	if fileName not in ["addGlobalColor"]:
-		print "Fatal error: could not find '%s or %s or %s'." % (path,p1,p2)
+		print("Fatal error: could not find '%s or %s or %s'." % (path,p1,p2))
 	raise FDKEnvError
 
 def runShellCmd(cmd):
 	try:
 		p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT).stdout
 		log = p.read()
-		return log
+		return str(log)
```

Comment:
that's wrong. `log` returned by subprocess is a bytes string. On py3, if you do `str(b"a")` without an `encoding` argument, you get its repr `"b'a'". You want to call `decode` method of bytes with the encoding that the subprocess command is using to print to stdout (probably ascii or utf-8, but you have to know).

Reply:
True, this is really wrong... but it works. To my defence: the whole file is a mess and needs a decent fix up...

will fix it

Reply:
agree

Reply:
fixed ac1494d1c3ba2a0fd99b13634b8aae217552ba92